src/common/AdaptiveStream.cpp

```diff
@@ -228,6 +228,10 @@ bool AdaptiveStream::download(const std::string& url,
     std::vector<char> bufferData(bufferSize);
     ssize_t totalReadBytes{0};
     bool isEOF{false};
+    std::string transferEncodingStr{
+        file.GetPropertyValue(ADDON_FILE_PROPERTY_RESPONSE_HEADER, "Transfer-Encoding")};
+    bool isChunked{!transferEncodingStr.empty() &&
+                   transferEncodingStr.find("hunked") != std::string::npos};
```

Comment:
is there a reason why you have kept `hunked` ? in documentation is specified `chunked`
for `transferEncodingStr` from rules we have to avoid specify the type of variable on the var name, thats why i change it

Reply:
@CastagnaIT to give more detail - I have seen it done this way for other implementations of libcurl. 
If you prefer we can do `bool isChunked{transferEncodingValue.find("chunked") != std::string::npos || transferEncodingValue.find("Chunked") != std::string::npos};` but I figure that the one test for "hunked" is a little less verbose

Reply:
its ok, yesterday there was internet down in italy many countries have been affected, after i forgot to answer..